### PR TITLE
Update graph.py and Simplifies assignment logic for G

### PR DESCRIPTION
**Pull request by metacritical**
Simplifies assignment logic for G

### CODE REVIEW

**Comment by tinyb0t** [comment]
Changes made in `tinygrad/`:
```
------------------------------------------------------------
files                             insertions       deletions
------------------------------------------------------------
tinygrad/graph.py                          1               1
------------------------------------------------------------
lines added in the tinygrad folder: 0
```

File: tinygrad/graph.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # **** debugging and graphing ****
 
-G = nx.DiGraph() if nx is not None else None
+G = nx.DiGraph()
```
Comment (jla524) [inline tinygrad/graph.py:17]: if networkx is not installed, this will fail with `AttributeError: 'NoneType' object has no attribute 'DiGraph'`

**Comment by jla524** [comment]
@geohot can we get rid of this try except logic, since networkx is in `setup.py`?
```
try:
  import networkx as nx  # type: ignore
except ImportError:
  nx = None  # graph won't work
```

File: tinygrad/graph.py (hunk 1 above)
Comment (metacritical) [reply tinygrad/graph.py:17]: If setup doesnt work we have a different problem also this is handled above.

**Pull request closed without merging**